arcgis-ios-sdk-samples/Geometry/Convex hull/README.metadata.json

```diff
@@ -1,9 +1,9 @@
 {
     "category": "Geometry",
-    "description": "Create a convex hull for a given set of points. The convex hull is a polygon with shortest perimeter that encloses a set of points. As a visual analogy, consider a set of points as nails in a board. The convex hull of the points would be like a rubber band stretched around the outermost nails.",
+    "description": "Create a convex hull for a given set of points. The convex hull is a polygon with shortest perimeter that encloses a set of points.",
     "ignore": false,
     "images": [
-        "image1.png"
+        "convex-hull.png"
```

Comment:
Hmm nice that you fix it here. But I also made those changes in #885 . We should use either one and ensure it includes all the fixes to make.

Reply:
It seems like #885 only involves README/metadata changes. Since we're updating the metadata now, we can fix the issues you listed in #885 during this effort as well.

Reply:
Sounds good, then I'll merge that PR into your current branch here!